build/en/docs/index.njk

```diff
@@ -192,30 +256,36 @@ rootClass: docs
     <p>
       Instead, use assets that are clean, and layer text elements on top using the element.
     </p>
+    <h2 id="Optimizing-your-images-videos">Optimizing your images &amp; videos</h2>
     <p>
-      <b>Note</b>: Version 1.0 of Web Stories for WordPress does not yet come with soft caption support for video. Support is coming shortly.
-    </p>
-    <h2 id="Optimizing-your-images-videos">Optimizing your images & videos</h2>
-    <p>
-      Since the editor doesn’t yet have the capability to automatically optimize video & image content for optimal load times, it’s a good idea to to invest into optimizing them in advance.
+      Since the editor doesn’t yet have the capability to automatically optimize video &amp; image content for optimal load times, it’s a good idea to to invest into optimizing them in advance.
```

Comment:
```suggestion
      Since the editor doesn’t yet have the capability to automatically optimize video &amp; image content for optimal load times, it’s a good idea to invest into optimizing them in advance.
```